### DISCUSSION TITLE
[dubbo-spi-extensions]Introduce new script routing in apache/dubbo-spi-extensions.

### DISCUSSION THREAD

**Discussion by songxiaosheng** [apache/dubbo#13834] [General]
The script routing provided in the repo has been out of date for several releases. We can consider introducing a new script routing mechanism for more convenient and flexible traffic routing support.

For security considerations, we can move the implementation to apache/dubbo-spi-extensions.

#13548

**Comment by AlbumenJ** [comment]
Track in https://github.com/apache/dubbo/discussions/13834
